test/functional/test-url-replacements.js

```diff
@@ -145,6 +146,78 @@ describe('UrlReplacements', () => {
     });
   });
 
+  it('should replace PAGE_LOAD_TIME', () => {
+    return expand('?sh=PAGE_LOAD_TIME').then(res => {
+      expect(res).to.match(/sh=\d+/);
+    });
+  });
+
+  it('should replace PAGE_LOAD_TIME if available within a delay', () => {
+    let win = {
+      'performance': {
+        'timing': {
+          'navigationStart': 100,
+          'loadEventStart': 0
+        }
+      },
+      'Object': Object
+    };
+
+    // When the delay does not result in any change.
+    urlReplacementsFor(win).expand('?sh=PAGE_LOAD_TIME&s').then(res => {
```

Comment:
These wouldn't work this way. Basically, in a test, you have to return the promise if you want it to be actually processed by the test.

That does mean that you may need to get a bit more granular. In this case you are doing two checks on promises - the both have to block the test returning. So you either have to split test in two or you can use our `all` call in `promise.js` which combines promises together.